pages/linux/expr.md

```diff
@@ -0,0 +1,23 @@
+# expr
+
+> Evaluate expressions and manipulate strings.
+
+- Get string length:
+
+`expr length {{string}}`
+
+- Evaluate logical or math expression with an operator('+','-','*','&','|' and etc.), special symbols should be escaped:
```

Comment:
I suppose the only way around this is to use two or three concrete examples rather than a single generic one. In any case, we can always improve the page later :)